### ISSUE TITLE
Check for issues in preprocessing

### ISSUE THREAD

**Issue by bethac07** [broadinstitute/superurop-broad-cimini#6]
@alicelucas recommended in checkin (Alice, please correct me if I'm wrong)

Check to make sure that your network DOES overfit with a small number of images; if it does NOT, then it implies that there is an issue with the preprocessing.

**Comment by alicelucas** [comment]
Yes! If it does not overfit on a small number of images (e.g., just 3 examples per class), then we either:
- are doing something wrong with the preprocessing (e.g. are we normalizing correctly?)
- are we using the right output layer / loss function? (is the range of the output layer consistent with what the loss function expects? in classification pay attention to sparse categorical entropy vs regular cross entropy,e tc)

**Comment by teresa-gao** [comment]
Next steps:

- one more attempt at getting a domain-specific backbone working (possibly with ResNet or some non-AlexNet architecture); and/or
- moving on to replicating the results of [PANet](https://github.com/kaixin96/PANet) and performing minimal tweaks to get things working with the Human Protein Atlas dataset
